browser/css/jssidebar.css

```diff
@@ -648,6 +648,27 @@ span.jsdialog.sidebar.ui-treeview-notexpandable {
 	grid-template-columns: 1fr;
 }
 
+#quickfind-container #quickfind-controls {
+	position: absolute;
+	top: 0;
+	right: 0;		
+}
+
+#quickfind-container #quickfind-controls button img {
+	width: var(--btn-img-size-s);
+	height: var(--btn-img-size-s);
```

Comment:
needs to be `--btn-img-size-m` instead to pair with its container that uses `--btn-size-m`. 

Additionally, We could maybe decrease the margin, but I don't want to add more specific quick find css rules, so for now let's keep it like that.

Reply:
Made the change, thanks for the review